code/game/objects/structures/tables_racks.dm

```diff
@@ -294,6 +299,59 @@
 	..()
 	return SECONDARY_ATTACK_CONTINUE_CHAIN
 
+/obj/structure/table/attack_hand_secondary(mob/user, list/modifiers)
+	. = ..()
+	if(!istype(user) || !user.can_interact_with(src) || isobserver(user))
+		return
```

Comment:
```suggestion
	if(!isliving(user) || !user.can_interact_with(src))
		return
```
Though AFAIK `attack_hand_secondary` is only ever invoked by humans so this maybe redundant (but safe, I guess)